lib/runtime/unknown_transaction_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/transaction"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUnknownTransactionErrors(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name        string
+		test        []byte
+		expErr      bool
+		expErrMsg   string
+		expValidity *transaction.Validity
+	}{
+		{
+			name:      "lookup failed",
+			test:      []byte{1, 1, 0},
+			expErrMsg: "lookup failed",
+			expErr:    true,
+		},
+	}
+
+	for _, c := range testCases {
+		t.Run(c.name, func(t *testing.T) {
```

Comment:
Pin the variable for for loop + parallel tests (otherwise it could just run all subtests with the last test case only)

```suggestion
	for _, c := range testCases {
		c := c
		t.Run(c.name, func(t *testing.T) {
```